receiver/redisreceiver/client_test.go

```diff
@@ -32,6 +33,10 @@ func newFakeClient() *fakeClient {
 }
 
 func (c fakeClient) delimiter() string {
+	if runtime.GOOS == "windows" {
+		return "\r\n"
+	}
```

Comment:
This caused a lot of issues due to calls to `strings.Split` not cleaning up the carriage return characters. It's only test code so safe to make this change here.